apps/testingaccessibility/src/pages/api/prices.ts

```diff
@@ -17,6 +18,8 @@ const pricesHandler = async (req: NextApiRequest, res: NextApiResponse) => {
 
       const couponId = coupon ? coupon : activeSaleCouponId
 
+      if (quantity > 100) throw new Error(`contact-for-pricing`)
```

Comment:
this throws an error, but really we just want to display a relevant message

the idea is that the dollar amounts over 100 units are very high and probably needs a conversation anyway